requirements-dev.in

```diff
@@ -1,5 +1,8 @@
 # pip-compile requirements-dev.in
 
+Django==3.1.8 # freeze version to the same as in production
+djangorestframework==3.12.2 # freeze version to the same as in production
```

Comment:
Why set here in addition to `requirements.in`?

Reply:
Ah well, the problem this was leading to is that requirements compiled in requirements-dev.txt pinned a different Django version, and then in reality you ended up working with a different Django version in development vs. in production. This makes sure we always work with the same Django version, plus any dev requirements relying on Django will have to work with the same version of Django as used in production.

In addition to this, I introduced one more change so that we prioritize production requirements on top of development requirements to make sure we always work with production dependencies even when developing.